src/app/clusters/basic/basic.cpp

```diff
@@ -221,6 +273,13 @@ void emberAfBasicClusterServerInitCallback(chip::EndpointId endpoint)
         status = Attributes::UniqueID::Set(endpoint, CharSpan(uniqueId, strlen(uniqueId)));
         VerifyOrdo(EMBER_ZCL_STATUS_SUCCESS == status, ChipLogError(Zcl, "Error setting Unique Id: 0x%02x", status));
     }
+
+    static bool attrAccessRegistered = false;
+    if (!attrAccessRegistered)
+    {
+        registerAttributeAccessOverride(&gBasicAttrAttrAccess);
```

Comment:
Why not put this inside `MatterBasicPluginServerInitCallback` instead, which is not called per-endpoint, so you don't need the guard?

Reply:
Where I can find the definition of ServerInitCallback and PluginServerInitCallback?

Reply:
I mean there different?

Reply:
The MatterClusterNamePluginServerInitCallback is called once, at stack bringup (modulo https://github.com/project-chip/connectedhomeip/issues/11352).  It's meant for cluster-singleton initialization.

emberAfClusterNameClusterServerInitCallback is called for every endpoint that gets enabled that has the cluster on it.  It's meant for initialization of per-endpoint state.

Reply:
OK, I will change it